lambdas/service/app.py

```diff
@@ -821,36 +825,84 @@ def get_order():
     return {'order': Plugin.load().service_config().order_config}
 
 
-@app.route('/manifest/files', methods=['GET'], cors=True)
-def start_manifest_generation():
-    """
-    Initiate and check status of a manifest generation job, returning a either a 301 or 302 response
-    redirecting to either the location of the manifest or a URL to re-check the status of the manifest job.
+manifest_filters = {
+    facet_name: schema.object(**{'is': schema.array(str)})
```

Comment:
Why so complicated? 

```suggestion
    facet_name: schema.object(is=schema.array(str))
```

Reply:
The complicated approach was needed because `is` is a python reserved word.
```
>>> def f(**params) -> bool:
...    print(params)    
    
>>> f(foo='bar')
{'foo': 'bar'}

>>> f(is='test')
  File "<input>", line 1
    f(is='test')
       ^
SyntaxError: invalid syntax

>>> f(**{'is': 'test'})
{'is': 'test'}
```

Reply:
Got it, that makes sense.